### PR TITLE
remove jsdelivr versioning of graphql-playground

### PR DESCRIPTION
**Pull request by dt665m**
This is a fix for #853

### CODE REVIEW

File: juniper/src/http/playground.rs [hunk 1]
```diff
@@ -23,7 +23,7 @@ pub fn playground_source(
   <title>GraphQL Playground</title>
   <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/graphql-playground-react/build/static/css/index.css" />
   <link rel="shortcut icon" href="//cdn.jsdelivr.net/npm/graphql-playground-react/build/favicon.png" />
-  <script src="//cdn.jsdelivr.net/npm/graphql-playground-react@1.7.27/build/static/js/middleware.js"></script>
+  <script src="//cdn.jsdelivr.net/npm/graphql-playground-react/build/static/js/middleware.js"></script>
```
Comment (tyranron) [inline juniper/src/http/playground.rs:26]: I don't think that removing the version at all is a good way to go, as it's the same as have `tokio = "*"` in `Cargo.toml`. You will never now when the things blew up on breaking change.

But!

We can just follow semver and specify `@1` version. So we catch up with recent updates but don't hit major version breakage.

File: juniper/src/http/playground.rs [hunk 2]
```diff
@@ -23,7 +23,7 @@ pub fn playground_source(
   <title>GraphQL Playground</title>
   <link rel="stylesheet" href="//cdn.jsdelivr.net/npm/graphql-playground-react/build/static/css/index.css" />
   <link rel="shortcut icon" href="//cdn.jsdelivr.net/npm/graphql-playground-react/build/favicon.png" />
```
Comment (tyranron) [inline juniper/src/http/playground.rs:25]: And it seems that here and in the line above we should specify `@1` version too.

**Review by tyranron — CHANGES_REQUESTED** [review]
@dt665m thanks for the contribution!

**Comment by Zireael13** [comment]
Sorry didn't see this PR before I created #856. As I noted there, just using the latest version (currently 1.7.27) won't work as that is the one that is broken. Need to pin it to 1.7.26 which I did in the other PR. Can move the changes over here if you'd like @tyranron

**Comment by dt665m** [comment]
> Sorry didn't see this PR before I created #856. As I noted there, just using the latest version (currently 1.7.27) won't work as that is the one that is broken. Need to pin it to 1.7.26 which I did in the other PR. Can move the changes over here if you'd like @tyranron

for some reason, leaving out the semver on jsdeliver resolves to a working version.  We're running off my PR here and it works.  🤷‍♂️

**Comment by Zireael13** [comment]
> for some reason, leaving out the semver on jsdeliver resolves to a working version. We're running off my PR here and it works. 🤷‍♂️

https://cdn.jsdelivr.net/npm/graphql-playground-react/build/static/js/middleware.js

Huh you are right. That is really odd. jsDelivr must somehow resolve to a past version if the file isn't present in the new one, I guess.

I guess we'll leave it up to the repo maintainers if they want to go with the versionless one or pinned to 1.7.26.

**Comment by LegNeato** [comment]
Generally I like explicit versioning as it helps reproducible builds as well as tracking down bugs. Thank you for the PR!

**Pull request closed without merging**